Sources/Sentry/SentryUIApplication.m

```diff
@@ -56,6 +56,105 @@ - (UIApplication *)sharedApplication
     return result;
 }
 
+- (NSArray<UIViewController *> *)relevantViewControllers
+{
+    NSMutableArray *result = [NSMutableArray array];
+
+    NSArray<UIWindow *> *windows = [self windows];
+    if ([windows count] == 0)
+        return nil;
+
+    for (UIWindow *window in windows) {
+        NSArray<UIViewController *> *vcs = [self relevantViewControllerFromWindow:window];
+        if (vcs != nil) {
+            [result addObjectsFromArray:vcs];
+        }
+    }
+
+    return result;
+}
+
+- (NSArray<UIViewController *> *)relevantViewControllerFromWindow:(UIWindow *)window
+{
+    UIViewController *rootViewController = window.rootViewController;
+    if (rootViewController == nil) {
+        return nil;
+    }
+
+    NSMutableArray<UIViewController *> *result = @[ rootViewController ].mutableCopy;
+    NSUInteger index = 0;
+
+    while (index < result.count) {
+        UIViewController *topVC = result[index];
+        // If the view controller is presenting another one, usually in a modal form.
+        if (topVC.presentedViewController != nil) {
+            [result replaceObjectAtIndex:index withObject:topVC.presentedViewController];
+            continue;
+        }
+
+        // The top view controller is meant for navigation and not content
+        if ([self isContainerViewController:topVC]) {
+            [result removeObjectAtIndex:index];
+            [result addObjectsFromArray:[self relevantViewControllerFromContainer:topVC]];
+            continue;
+        }
+
+        UIViewController *relevantChild = nil;
+        for (UIViewController *childVC in topVC.childViewControllers) {
+            // Sometimes a view controller is used as container for a navigation controller
+            // If the navigation is occupating the whole view controller we will consider this the
+            // case.
+            if ([self isContainerViewController:childVC]
+                && CGRectEqualToRect(childVC.view.frame, topVC.view.bounds)) {
```

Comment:
Why are you comparing `frame` and `bounds`? `bounds` will always have an origin of `{0,0}`, but if a vc has multiple child vcs layed out in different locations, their `frame` origins will be nonzero in x or y for some of them; there might not be any at `{0,0}`.

Reply:
Because I want to compare the child view controller view frame to the bounds of its parent view controller view. 

This check is a big stretch, but im my experience it happens from time to time.